### PR TITLE
package/python-dbus-next: include new dbus-next software

### PR DESCRIPTION
**Pull request by mhungerford-cpi**
python-dbus-next is a Python library for DBus that aims to be a fully featured high level library primarily geared towards integration of applications into Linux desktop and mobile environments.

* Zero dependencies and pure Python 3.
* Support for multiple IO backends including asyncio and GLib
* Nonblocking IO suitable for GUI development.
* Target the latest language features of Python
* Complete implementation of the DBus type system

### CODE REVIEW

**Review by bwei-cpi — APPROVED** [review]
looks good

File: package/python-dbus-next/python-dbus-next.hash [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# md5, sha256 from https://pypi.org/pypi/uvloop/json
```
Comment (jhart-cpi) [inline package/python-dbus-next/python-dbus-next.hash:1]: this comment doesn't seem right?

**Review by jhart-cpi — CHANGES_REQUESTED** [review]
`package/Config.in` has to be modified for buildroot to include this library in a build.

File: package/python-dbus-next/python-dbus-next.hash (hunk 1 above)
Comment (mhungerford-cpi) [reply package/python-dbus-next/python-dbus-next.hash:1]: ah, old comment, will fix

**Comment by mhungerford-cpi** [comment]
> `package/Config.in` has to be modified for buildroot to include this library in a build.

The emb-platform/os/buildroot/ucb/buildroot_defconfig is where we enable these optional libraries, not the main buildroot conf.

**Review by jhart-cpi — APPROVED** [review]

**Review by dleeds-cpi — APPROVED** [review]

**Pull request merged**